README.md

```diff
@@ -16,53 +17,116 @@ When you are done:
 - [Max Value](https://repl.it/@Admin7/maxvalue)
 ```python
 # copy & paste your solution here
+def return_max(aList):
+  if len(aList) == 0:
+    return None
+  return max(aList)
 ```
 
 - [Min Value](https://repl.it/@Admin7/minvalue)
 ```python
 # copy & paste your solution here
+def return_min(aList):
+  if len(aList) == 0:
+    return None
+  return min(aList)
 ```
 
 - [Extend](https://repl.it/@Admin7/extendlist)
 ```python
 # copy & paste your solution here
+def return_extended(aList, iter):
+  for item in iter:
+    aList.append(item)
+  return aList
 ```
 
 - [Extend](https://repl.it/@Admin7/creatematrix)
 ```python
 # copy & paste your solution here
+def create_matrix(h, w):
+  # a list to hold the list
+  aList = []
```

Comment:
I would reconsider the naming here! If another programmer picked this up, how quickly would they be able to identify which list represents a matrix and which represents a row? Probably not too quickly! `matrix` and `col` would probably be better variable names here.